test/abort/test-abort-unhandled-rejection.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const spawn = require('child_process').spawn;
+const node = process.execPath;
+
+if (process.argv[2] === 'child') {
+  Promise.reject(new Error('child error'));
+} else {
+  run('', null);
+  if (!common.isWindows) {
```

Comment:
Should we `common.skip()` on Windows, or perhaps assert that the flag is rejected?